scalableminds/webknossos
# Check wK write permission before organization creation

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create an organization when the datastore doesn't has write permission => should not work
- create an organization when the datastore has write permission => should still work

### Issues:
- fixes #4455 

------
